ArgoTests/JSON/types.json

```diff
@@ -11,6 +11,7 @@
     "hello",
     "world"
   ],
+  "string_array_opt": null,
```

Comment:
It's being decoded in the `TestModel` decode function. then we're testing that it's nil in the TypeTests file. Should we add a test that gets the `Decoded<TestModel>` and checks if that is `.MissingKey()`?

What we have now should be testing that issue.